sites/public/pages/index.tsx

```diff
@@ -63,14 +56,7 @@ export default function Home(props: IndexProps) {
           {alertInfo.alertMessage}
         </AlertBox>
       )}
-      <Hero
-        title={heroTitle}
-        buttonTitle={t("welcome.seeRentalListings")}
-        buttonLink="/listings"
-        allApplicationsClosed={
-          !props.listings.some(listingOpen) && !props.listings.some(openDateState)
-        }
-      />
```

Comment:
This piece has been updated on Alameda for some time now, since the current logic doesn't account for listings with no application due date. I'm not sure we really need this check here, and it gives us one less route where we're querying listings.